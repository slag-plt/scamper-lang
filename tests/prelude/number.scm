(number? 56)
(number? "56")
(number? 56.2)
(number? (/ 56 2))
(number? 56.0)