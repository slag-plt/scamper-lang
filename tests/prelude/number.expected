#t
#f
#t
#t
#t
